src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -3272,90 +3273,110 @@ public DynamicJsonValue GetLogDetails(ClusterOperationContext context, int max =
+                }
+                else
+                {
+                    var tempBuffersDiskSpaceResult = DiskUtils.GetDiskSpaceInfo(environment.Environment.Options.TempPath.FullPath, driveInfo?.TempPath);
+                    if (tempBuffersDiskSpaceResult != null)
                     {
-                        yield return new Client.ServerWide.Operations.MountPointUsage
+                        var tempBuffersUsage = new Client.ServerWide.Operations.MountPointUsage
                         {
                             Name = environment.Name,
                             Type = environment.Type.ToString(),
                             UsedSpaceByTempBuffers = sizeOnDisk.TempBuffersInBytes,
-                            DiskSpaceResult = new Client.ServerWide.Operations.DiskSpaceResult
-                            {
-                                DriveName = tempBuffersDiskSpaceResult.DriveName,
-                                VolumeLabel = tempBuffersDiskSpaceResult.VolumeLabel,
-                                TotalFreeSpaceInBytes = tempBuffersDiskSpaceResult.TotalFreeSpace.GetValue(SizeUnit.Bytes),
-                                TotalSizeInBytes = tempBuffersDiskSpaceResult.TotalSize.GetValue(SizeUnit.Bytes)
-                            }
+                            DiskSpaceResult = FillDiskSpaceResult(tempBuffersDiskSpaceResult)
                         };
+                        var tempBufferIoStatsResult = Server.DiskStatsGetter.Get(driveInfo?.TempPath.DriveName);
+                        if(tempBufferIoStatsResult != null)
+                            tempBuffersUsage.IoStatsResult = FillIoStatsResult(ioStatsResult);  
+
+                        yield return tempBuffersUsage;
                     }
                 }
             }
 
             yield return usage;
         }
 
+        private static Client.ServerWide.Operations.DiskSpaceResult FillDiskSpaceResult(Sparrow.Server.Utils.DiskSpaceResult journalDiskSpaceResult)
+        {
+            return new Client.ServerWide.Operations.DiskSpaceResult
+            {
+                DriveName = journalDiskSpaceResult.DriveName,
+                VolumeLabel = journalDiskSpaceResult.VolumeLabel,
+                TotalFreeSpaceInBytes = journalDiskSpaceResult.TotalFreeSpace.GetValue(SizeUnit.Bytes),
+                TotalSizeInBytes = journalDiskSpaceResult.TotalSize.GetValue(SizeUnit.Bytes)
+            };
+        }
+
+        private static IoStatsResult FillIoStatsResult(DiskStatsResult ioStatsResult)
+        {
+            return new IoStatsResult
+            {
+                IoReadOperations = ioStatsResult.IoReadOperations, 
+                IoWriteOperations = ioStatsResult.IoWriteOperations,
+                ReadThroughputInKb = ioStatsResult.ReadThroughput.GetValue(SizeUnit.Kilobytes),
+                WriteThroughputInKb = ioStatsResult.WriteThroughput.GetValue(SizeUnit.Bytes),
```

Comment:
For Read we say `.GetValue(SizeUnit.Kilobytes)` and here `.GetValue(SizeUnit.Bytes)`. 
Which is it?

Reply:
I meant it to be `Kilobytes`
done